docs/src/selectors.md

```diff
@@ -1167,6 +1167,32 @@ document
 
 If a selector needs to include `>>` in the body, it should be escaped inside a string to not be confused with chaining separator, e.g. `text="some >> text"`.
 
+## Parent selector
```

Comment:
I think this now breaks "Intermediate matches" being a subitem of "Chaining selectors". I'd put this whole section right above the "Chaining selectors".

Reply:
Ow yes indeed, thx! Will fix this

Reply:
Done!